aspnetcore/signalr/index.md

```diff
@@ -25,3 +25,4 @@ uid: signalr/index
 * [MessagePack Hub Protocol](xref:signalr/messagepackhubprotocol)
 * [Streaming](xref:signalr/streaming)
 * [Differences between SignalR versions](xref:signalr/version-differences)
+* [JavaScript API](/javascript/api/)
```

Comment:
I suggest changing the link to the following:

```
* [JavaScript API reference](/javascript/api/?view=signalr-js-latest)
```

This should also probably move under the **Clients** node. At least, that's where we decided to put the associated **JavaScript client** doc.